data/countries/MX.yaml

```diff
@@ -1,5 +1,19 @@
 holidays:
   # @attrib https://en.wikipedia.org/wiki/Public_holidays_in_Mexico
+  #
+  # Bank Holidays:
```

Comment:
This is a bit verbose but my goal here is to help myself (or someone else) in the future to be able to find the 2020 dates when they are published. The pattern of which dates are banking holidays has not changed in the past three years.

Reply:
Great. Then let's use the pattern until it may change...